### ISSUE TITLE
[archeology] can't reach the artefact

### ISSUE THREAD

**Issue by Damvs** [HellscreamWoW/Tracker#3006]
[IMG]http://img15.hostingpics.net/pics/988006WoWScrnShot101615175716.jpg[/IMG]

**Comment by Damvs** [comment]
The artefact spawn at the topfloor

**Comment by silverjade** [comment]
Yeah, I've noticed Artifacts seem to come from the sky or so it appears. Since if there is anything above you when you discover the artifact fragments, it will spawn either on top of it or in it.
